apps/dashboard/components/batch-payments/index.tsx

```diff
@@ -0,0 +1,280 @@
+            width: "100%",
+          }}
+        >
+          <ModalClose variant="plain" />
+          <Typography
+            component="h2"
+            id="modal-callback"
+            level="h4"
+            textColor="inherit"
+            fontWeight="lg"
+            mb={1}
+          >
+            {modalDetails.heading}
+          </Typography>
+          <p>{modalDetails.message}</p>
+        </Sheet>
+      </Modal>
+
+      {csvData.length > 0 && paymentDetails && paymentDetails.walletDetails ? (
+        <>
+          <Box
+            sx={{
+              marginBottom: "1em",
+              display: "flex",
+              flexDirection: "row",
+              width: "100%",
+              gap: "1em",
+            }}
+          >
+            <DetailsCard>
+              <Details
+                label="Total Amount"
+                value={`${String(
+                  paymentDetails.totalAmount,
+                )} ${displayCurrencyBatchPayments({
+                  walletCurrency: paymentDetails.walletDetails.walletCurrency,
+                })}`}
+              />
+              <Details
+                label="Wallet Type"
+                value={paymentDetails.walletDetails.walletCurrency}
+              />
+              <Details label="Wallet Id" value={paymentDetails.walletDetails.id} />
+              <Details
+                label="Wallet Balance"
+                value={displayWalletBalanceBatchPayments({
+                  amount: paymentDetails.walletDetails.balance,
+                  walletCurrency: paymentDetails.walletDetails.walletCurrency,
+                })}
+              />
+              <Button onClick={processPayments} loading={processPaymentLoading}>
+                Confirm Payment
+              </Button>
+            </DetailsCard>
+            <FileUpload
+              processCsvLoading={processCsvLoading}
+              file={file}
+              setFile={setFile}
+            />
+          </Box>
+          <BatchPaymentsList
+            processedList={csvData}
+            walletCurrency={paymentDetails.walletDetails.walletCurrency}
+          ></BatchPaymentsList>
+        </>
+      ) : (
+        <>
+          <FileUpload
+            processCsvLoading={processCsvLoading}
+            file={file}
+            setFile={setFile}
+          />
+          <Card
+            sx={{
+              margin: "1em auto",
+              padding: "1em",
+            }}
+          >
+            {`Please upload a CSV file containing the following columns: "username" (the
+            recipient of the bitcoin), "dollars/sats" (this column will indicate the amount;
+            the header of this column will determine which wallet to use. If the header is
+            "dollars," your USD wallet will be utilized, and the amount will be in US dollars. If
+            "sats" is used, your BTC wallet will be employed, and the amount will be in
+            sats), and "memo" (optional)`}
```

Comment:
I don't think this is intuitive

I might typically want to use my bitcoin wallet for a dollar denominated payment

I think a more intuitive way would be to have an optional walletId to pay from, and if this is not provided, then the defaultWalletId should be used to make the payment

then we can have 2 columns, one called amount, and the other called denomination that can either be USD or BTC

also, we should ensure that the CSV we are using here maps the CSV we have for galoy-cli. I don't think that is the case currently. Maybe the galoy-cli may need some iteration also.